packages/terra-date-input/translations/fr-FR.json

```diff
@@ -2,7 +2,7 @@
   "Terra.date.input.optional": "(facultatif)",
   "Terra.date.input.required": "Requis",
   "Terra.date.input.monthLabel": "Mois",
-  "Terra.date.input.monthPlaceholder": "Mois",
+  "Terra.date.input.monthPlaceholder": "Sélectionner",
```

Comment:
Is there a reason why French doesn't have the `-` before and after the string like other languages?

Reply:
I don't know why. I copied these translations directly from the [form-select](https://github.com/cerner/terra-core/blob/main/packages/terra-form-select/translations/fr-FR.json#L4).